### PR TITLE
PING-3168 Chat detail page not loading

### PR DESCRIPTION
**Pull request by agit26milan**

### CODE REVIEW

File: src/hooks/core/websocket/useSimpleWebsocketHooks.tsx [hunk 1]
```diff
@@ -86,7 +86,6 @@ const useSimpleWebsocket = (url, protocol = null) => {
   };
 
   React.useEffect(() => {
-    setupWebsocket();
```
Comment (fajarism) [inline src/hooks/core/websocket/useSimpleWebsocketHooks.tsx:89]: mas @agit26milan why is this deleted?

**Review by fajarism — COMMENTED** [review]
please check my comment

**Review by arilukmanp — APPROVED** [review]

**Review by fajarism — APPROVED** [review]
